back/app/policies/user_policy.rb

```diff
@@ -63,7 +63,15 @@ def destroy?
     record.id == user&.id || (user&.active? && user&.admin?)
   end
 
-  def blocked_count
+  def block?
+    index?
```

Comment:
Just one product-related question to keep in mind. Does it mean that one user with manager permissions to one project can block all the admins and other managers? But maybe it's not a big deal.

Reply:
True. Maybe we should add a check that user to be blocked is not an admin. I'll ask for clarification.

But, like you say, not an urgent issue, and we can cover it in a separate PR if needed.

Reply:
Makes sense.